src/commonMain/kotlin/exchange.dydx.abacus/validator/trade/TradeMarketOrderInputValidator.kt

```diff
@@ -141,7 +141,8 @@ internal class TradeMarketOrderInputValidator(
             )
         }
 
-        val summary = parser.asNativeMap(trade["summary"]) ?: return null
+        // summary can be empty even though there's input e.g. leverage input is set to the same as current position leverage
+        val summary = parser.asNativeMap(trade["summary"])?.takeIf { it.isNotEmpty() } ?: return null
```

Comment:
i hope the typed version avoids this t_t? but it's also where summary is set

Reply:
It should be fine.  The typed version has `trade.summary` being either null or filled with data.